.browserslistrc

```diff
@@ -6,66 +6,74 @@ last 1 safari version
 node 14
 
 # Default/Fallback
-# `npx browserslist --mobile-to-desktop "> 0.5%, last 2 versions, Firefox ESR, not dead, not IE 11"` when the last major is released.
-# Manually downgrading to ios_saf 12.4 for iPhone 6 and webpack 4 support.
+# `npx browserslist --mobile-to-desktop "> 0.2%, last 2 versions, Firefox ESR, not dead"` when the last major is released.
 # On update, sync references where "#stable-snapshot" is mentioned in the codebase.
 [stable]
-and_chr 91
-and_ff 89
-and_qq 10.4
-and_uc 12.12
-android 91
-baidu 7.12
-chrome 90
-edge 91
-firefox 78
-# 12.4 but 12.2-12.5 are treated equally in caniuse-lite.
-# Though caniuse-lite does not supporting finding an exact version in a range which is why `12.4` would result in "Unknown version 12.4 of ios_saf"
-ios_saf 12.2
+and_chr 122
+and_chr 121
+and_ff 123
+and_ff 122
+and_qq 14.9
+and_uc 15.5
+android 122
+android 121
+chrome >=109
+edge >=120
+firefox 123
+firefox 122
+firefox 115
+ios_saf >= 15.6
+kaios 3.0-3.1
 kaios 2.5
 op_mini all
-op_mob 73
-opera 76
-safari 14
-samsung 13.0
+op_mob 73 # The op_mob 80 is unknown, so downgrading to 73
+opera >=106
+safari >=15.6
+samsung 23
+samsung 22
 
-# Same as `stable` but with IE 11
 [legacy]
 ie 11
-and_chr 91
-and_ff 89
-and_qq 10.4
-and_uc 12.12
-android 91
-baidu 7.12
-chrome 90
-edge 91
-firefox 78
-ios_saf 12.2
+and_chr 122
+and_chr 121
+and_ff 123
+and_ff 122
+and_qq 14.9
+and_uc 15.5
+android 122
+android 121
+chrome >=109
+edge >=120
+firefox 123
+firefox 122
+firefox 115
+ios_saf >= 15.6
+kaios 3.0-3.1
 kaios 2.5
 op_mini all
-op_mob 73
-opera 76
-safari 14
-samsung 13.0
+op_mob 73 # The op_mob 80 is unknown, so downgrading to 73
```

Comment:
It does work in Base UI: https://github.com/mui/base-ui/blob/master/.browserslistrc
Have you tried updating caniuse-lite to the latest version?

Reply:
Let me try

Reply:
@michaldudak It really because of `caniuse-lite`. However, I don't see any usage of "caniuse-lite" in any of the package.json, so I make it work with `pnpm up "caniuse-lite"`. Is it the correct way to upgrade implicit deps? cc @Janpot

Reply:
> Is it the correct way to upgrade implicit deps?

👍  Probably also want to run `pnpm dedupe` after. But I see no problem with this.

Reply:
@siriwatknp Actually, it seems to be
```
px browserslist@latest --update-db
```